Benchmarks are broken

Benchmarks for other bindings are likely broken as well. It would be great if they were included in CI.

```text
┌─[tautvydas][kagamin][±][main ✓][~/Git/c-kzg-4844/bindings/rust]
└─▪ cargo bench
   Compiling c-kzg v0.1.0 (/mnt/home/Git/c-kzg-4844/bindings/rust)
error[E0308]: mismatched types
   --> benches/kzg_benches.rs:56:61
    |
56  |                         .verify_aggregate_kzg_proof(&blobs, &kzg_commitments, &kzg_settings)
    |                          --------------------------         ^^^^^^^^^^^^^^^^ expected slice, found struct `Vec`
    |                          |
    |                          arguments to this function are incorrect
    |
    = note: expected reference `&[Bytes48]`
               found reference `&Vec<c_kzg::KZGCommitment>`
note: associated function defined here
   --> /mnt/home/Git/c-kzg-4844/bindings/rust/src/lib.rs:205:12
    |
205 |     pub fn verify_aggregate_kzg_proof(
    |            ^^^^^^^^^^^^^^^^^^^^^^^^^^

For more information about this error, try `rustc --explain E0308`.
error: could not compile `c-kzg` due to previous error
```

--- a/bindings/rust/benches/kzg_benches.rs
+++ b/bindings/rust/benches/kzg_benches.rs
@@ -40,10 +40,10 @@ pub fn criterion_benchmark(c: &mut Criterion) {
             |b, blobs| b.iter(|| KZGProof::compute_aggregate_kzg_proof(blobs, &kzg_settings)),
         );
 
-        let kzg_commitments: Vec<KZGCommitment> = blobs
+        let kzg_commitments: Vec<Bytes48> = blobs
             .clone()
             .into_iter()
-            .map(|blob| KZGCommitment::blob_to_kzg_commitment(blob, &kzg_settings))
+            .map(|blob| KZGCommitment::blob_to_kzg_commitment(blob, &kzg_settings).to_bytes())
             .collect();
         let proof = KZGProof::compute_aggregate_kzg_proof(&blobs, &kzg_settings).unwrap();
 
@@ -53,7 +53,11 @@ pub fn criterion_benchmark(c: &mut Criterion) {
             |b, blobs| {
                 b.iter(|| {
                     proof
-                        .verify_aggregate_kzg_proof(&blobs, &kzg_commitments, &kzg_settings)
+                        .verify_aggregate_kzg_proof(
+                            &blobs,
+                            kzg_commitments.as_slice(),
+                            &kzg_settings,
+                        )
                         .unwrap()
                 })
             },

--- a/bindings/rust/src/lib.rs
+++ b/bindings/rust/src/lib.rs
@@ -434,10 +434,7 @@ mod tests {
 
             for (i, blob) in blobs.into_iter().enumerate() {
                 let commitment = KZGCommitment::blob_to_kzg_commitment(blob, &kzg_settings);
-                assert_eq!(
-                    commitment.as_hex_string().as_str(),
-                    expected_commitments[i]
-                );
+                assert_eq!(commitment.as_hex_string().as_str(), expected_commitments[i]);
             }
         }
     }
@@ -468,12 +465,7 @@ mod tests {
             let y_bytes = Bytes32::from_bytes(&hex::decode(y_hex).unwrap()).unwrap();
 
             assert!(kzg_proof
-                .verify_kzg_proof(
-                    commitment,
-                    z_bytes,
-                    y_bytes,
-                    &kzg_settings
-                )
+                .verify_kzg_proof(commitment, z_bytes, y_bytes, &kzg_settings)
                 .unwrap());
         }
     }

